implementations/rust/ockam/ockam_command/src/tcp/connection/create.rs

```diff
@@ -38,32 +40,23 @@ pub struct CreateCommand {
 
 impl CreateCommand {
     pub fn run(self, options: CommandGlobalOpts) {
-        let cfg = &options.config;
-        let node = extract_address_value(&self.node_opts.from).unwrap_or_else(|_| "".to_string());
-        let port = cfg.get_node_port(&node).unwrap();
-
-        connect_to(port, (self, options.clone()), create_connection);
+        node_rpc(run_impl, (options, self))
     }
 }
 
-pub async fn create_connection(
-    ctx: Context,
-    (cmd, opts): (CreateCommand, CommandGlobalOpts),
-    mut base_route: Route,
-) -> anyhow::Result<()> {
-    let resp: Vec<u8> = match ctx
+async fn run_impl(
+    ctx: ockam::Context,
+    (opts, cmd): (CommandGlobalOpts, CreateCommand),
+) -> crate::Result<()> {
+    let node = extract_address_value(&cmd.node_opts.from)?;
+    let port = opts.config.get_node_port(&node)?;
+    let mut base_route = route![(TCP, format!("localhost:{}", port))];
+    let resp: Vec<u8> = ctx
         .send_and_receive(
             base_route.modify().append(NODEMANAGER_ADDR),
             api::create_tcp_connection(&cmd)?,
         )
-        .await
-    {
-        Ok(sr_msg) => sr_msg,
-        Err(e) => {
-            eprintln!("Wasn't able to send or receive `Message`: {}", e);
-            std::process::exit(exitcode::IOERR);
-        }
-    };
+        .await?;
 
     let (response, TransportStatus { payload, .. }) = api::parse_transport_status(&resp)?;
```

Comment:
You have to replace all this and use an `Rpc` instance. Check out [this as a reference](https://github.com/build-trust/ockam/blob/develop/implementations/rust/ockam/ockam_command/src/secure_channel/delete.rs#L147-L150).